kolibri/plugins/bloompub_viewer/assets/src/views/BloomPubRendererIndex.vue

```diff
@@ -162,58 +156,21 @@
         this.loading = false;
         this.$emit('error', err);
       });
-      let storageUrl = this.defaultFile.storage_url;
-      if (!this.isBloom) {
-        // In the case that this is being routed via a remote URL
-        // ensure we preserve that for the zip endpoint.
-        const url = new URL(this.defaultFile.storage_url, window.location.href);
-        const baseurl = url.searchParams.get('baseurl');
-        storageUrl = urls.zipContentUrl(
-          this.defaultFile.checksum,
-          this.defaultFile.extension,
-          this.entry,
-          baseurl ? encodeURIComponent(baseurl) : undefined,
-        );
-      }
 
       this.hashi.initialize(
         (this.extraFields && this.extraFields.contentState) || {},
         this.userData,
-        storageUrl,
+        this.defaultFile.storage_url,
         this.defaultFile.checksum,
       );
       this.$emit('startTracking');
-      if (!this.isBloom) {
-        this.pollProgress();
-      }
     },
     beforeDestroy() {
       if (this.timeout) {
         clearTimeout(this.timeout);
       }
       this.$emit('stopTracking');
     },
-    methods: {
-      recordProgress() {
```

Comment:
and removing this is because the progress tracking isn't working, and we're fixing that in follow up?

Reply:
Because this method is only used for periodic polling of progress recording - whereas all progress tracking is driven by events from the bloom player for this (this was copy pasted from the HTML5 renderer).